docs/src/pages/docs/rag/vector-databases.mdx

```diff
@@ -139,38 +139,93 @@ chunks.map(chunk => ({ text: chunk.text }))
 
 ## Using Vector Storage
 
-Once initialized, all vector stores share the same interface for creating indexes, upserting embeddings, and querying:
+Once initialized, all vector stores share the same interface for creating indexes, upserting embeddings, and querying.
+
+### Creating Indexes
+
+Before storing embeddings, you need to create an index with the appropriate dimension size for your embedding model:
 
 ```ts filename="store-embeddings.ts" showLineNumbers copy
-// 1. Create an index (dimension = 1536 for text-embedding-3-small)
+// Create an index with dimension 1536 (for text-embedding-3-small)
 await store.createIndex('my-collection', 1536);
 
-// 2. Store embeddings with metadata
+// For other models, use their corresponding dimensions:
+// - text-embedding-3-large: 3072
+// - text-embedding-ada-002: 1536
+// - cohere-embed-multilingual-v3: 1024
+```
+
+The dimension size must match the output dimension of your chosen embedding model. Common dimension sizes are:
+- OpenAI text-embedding-3-small: 1536 dimensions
+- OpenAI text-embedding-3-large: 3072 dimensions
+- OpenAI text-embedding-ada-002: 1536 dimensions
```

Comment:
Let's not mention this one, it seems to be strictly inferior to `text-embedding-3-small`